pyproject.toml

```diff
@@ -22,10 +22,9 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
     "Typing :: Typed",
 ]
-dependencies = [
-]
 description = "Code and training materials for predicting short-term hospital bed capacity using real-time data"
 dynamic = [
+    "dependencies",
```

Comment:
This specifies the `dependencies` entry will be set dynamically elsewhere in file.